test/stream.browser.spec.ts

```diff
@@ -9,6 +12,14 @@ function setup (): { peerConnection: RTCPeerConnection, datachannel: RTCDataChan
   return { peerConnection, datachannel, webrtcStream }
 }
 
+function generatePbByFlag (flag?: pb.Message_Flag) {
```

Comment:
nit: could you add a return type here?

Reply:
Thanks for the reminder, the return type has been added

but CI is failing，I did not find the same error when I run the CI script locally

Reply:
I re-ran CI and it passed, might have been an issue of a cached dep (need to investigate).